pkg/operator/ceph/controller/spec.go

```diff
@@ -67,6 +67,25 @@ const (
 	startupProbeFailuresDaemonOSD int32 = 12 * 60
 )
 
+// mds probe constants
```

Comment:
```suggestion
```
comments below explains the const

Reply:
Why need a separate const variables thats why added it